vocode/streaming/models/vector_db.py

```diff
@@ -16,6 +16,6 @@ class VectorDBConfig(TypedModel, type=VectorDBType.BASE.value):
 
 class PineconeConfig(VectorDBConfig, type=VectorDBType.PINECONE.value):
     index: str
-    api_key: Optional[str]
-    api_environment: Optional[str]
+    api_key: Optional[str] = None
```

Comment:
nit: can we not make this change as part of this PR (this means that folks can run this without checking out your vocode fork / updating their code at all)

seems simple enough to just include a couple of `None`s in the vector DB config